prowler/lib/outputs/utils.py

```diff
@@ -5,7 +5,7 @@ def unroll_list(listed_items: list, separator: str = "|"):
             if not unrolled_items:
                 unrolled_items = f"{item}"
             else:
-                unrolled_items = f"{unrolled_items} {separator} {item}"
```

Comment:
I think we want to leave this for the `|` but not for the `,`, could you please adapt it?

Reply:
Done!